brightnessd.c

```diff
@@ -1017,6 +1030,80 @@ static uint8_t event_loop(struct Tglobalstate *pglobalstate, struct Txcb *pxcb,
     }
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// parse_uint8_t
```

Comment:
Nitpicking: The other functions have `()` appended in the header. Also applies to the other new functions below.

Reply:
Yes, obviously my screen size was too small to notice ;) Fixed.